README.md

```diff
@@ -219,18 +235,73 @@ void main() {
 }
 ```
 
-This can be run using:
+You can launch this app directly from VS Code, or by running:
 
 ```bash
 flutter run -t path/to/my_scenes.stager_app.g.dart
 ```
 
-You can launch to a specific scene by providing the name of the scene as an argument:
+If your Stager app consists of multiple Scenes, you can launch to a specific scene by providing the name of the scene as an argument:
 
 ```bash
 flutter run -t path/to/my_scenes.stager_app.g.dart --dart-define='Scene=No Posts'
 ```
 
+### Adding your own environment controls
+
+Stager's control panel comes with a generally useful set of controls that enable you to toggle dark mode, adjust text scale, etc. However, it is very likely that your app has unique environment properties that would be useful to adjust. To support this, Scenes have an overridable `environmentControlBuilders` property which allows you to add custom widgets to the default set of environment manipulation controls.
```

Comment:
"environment" is what we assume they'll do with it but doesn't have to right? Maybe just "customControlBuilders" or "additionalControlBuilders"?